### PR TITLE
Add feature of event ticket printing in event_barcode

### PR DESCRIPTION
**Pull request by dbh-odoo**
Description of the issue/feature this PR addresses:
Task: https://www.odoo.com/web?#id=26779&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad: https://pad.odoo.com/p/r.683e502045b3a16ae7d4b119f97c530f

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mba-odoo** [comment]
LGTM

File: odoo/addons/base/models/ir_actions.py [hunk 1]
```diff
@@ -82,14 +82,15 @@ def _get_eval_context(self, action=None):
         }
 
     @api.model
-    @tools.ormcache('frozenset(self.env.user.groups_id.ids)', 'model_name')
-    def get_bindings(self, model_name):
+    @tools.ormcache('frozenset(self.env.user.groups_id.ids)', 'model_name', 'debug')
+    def get_bindings(self, model_name, debug=False):
```
Comment (RomainLibert) [inline odoo/addons/base/models/ir_actions.py]: Why do we need to add this debug attribute?

File: addons/web/static/src/js/report/report.editor.js [hunk 2]
```diff
@@ -13,11 +13,23 @@ var trusted_origin = utils.build_origin(trusted_protocol, trusted_host);
...
 // Patch the editor's behavior when it is launched inside an iframe.
 if (window.self !== window.top) {
-    $(document.body).addClass('o_in_iframe'); //  in order to apply css rules
-
+    editor.EditorMenuBarEdit.include({
+        start: function () {
+            return this._super.apply(this, arguments).then(function () {
+                // hide web_editor-toolbars if record is in readonly mode
+                // due to edit button available on the control panel of view
+                this.$('body').removeClass('editor_enable');
```
Comment (RomainLibert) [inline addons/web/static/src/js/report/report.editor.js]: Do we have to hide the whole toolbar, couldn't we just hide the buttons?

File: odoo/addons/base/models/ir_actions.py (hunk 1 above)
Comment (pga-odoo) [reply odoo/addons/base/models/ir_actions.py]: `base.group_no_one` is special group which activated on debug mode and mostly we manage it manually like this: 
https://github.com/odoo/odoo/blob/master/odoo/models.py#L1150

get_bindings is works well with all groups except `base.group_no_one` and it is not calculated again due to ormcache, so we need to pass/process it manually.
Exactly like menus: https://github.com/odoo/odoo/blob/master/odoo/addons/base/models/ir_ui_menu.py#L78

**Comment by tde-banana-odoo** [comment]
Task canceled.

**Pull request closed without merging**